script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -926,7 +928,8 @@ A final thrust with no other follow-up.
 [sha:229dfd5f41eb8cec462d6b1bf7ba71d085f5cf13]{
 -- Page 346, Offset 36281.
 -- ノエルという存在が打ち出せる極点を前に、彼女は銃身を回転させ、杭打ち機をセットする。
-Before the existence, formerly known as Noel, sends out her final strike, Ciel rotates the gun barrel and sets up her pile-driver.
+// all other instances of 極点 i saw seem to be 'focality', so just keeping it consistent here
```

Comment:
yeah this one is used a bunch in the vlov fight and i think focality is what we settled on